doc/admin-guide/logging/formatting.en.rst

```diff
@@ -295,6 +300,33 @@ ssh            essh
 cssh           ecssh
 ============== ===================
 
+It is also possible to log all of the headers in a transaction message with a
+single field.  For each original original field, there is a variant which ends in
+``ah`` rather than ``h``, as shown here:
+
+============== ===================
+Original Field All Headers Variant
+============== ===================
+cqh            cqah
+pqh            pqah
+psh            psah
+ssh            ssah
+cssh           cssah
+============== ===================
+
+No particular header is specified when using these variants, for example::
+
+    Format = '%<cqah>'
+
+The output generated by these fields has the pattern::
+
+    {{{tag1}:{value1}}{{tag2}:{value2}}...}
```

Comment:
I wonder if it might be better as
```
{{tag1}:{value1},{tag2}:{value2}, ...}
```
which would be more JSON like. Or even
```
[["tag1","value1"],["tag2","value2"],...]
```

Reply:
If we are going to use " (or ' ) to mark both the beginning and the end, then we need to add escaping to the contents.  Unless we are sure logs with these fields will never be machine-parsed.  Dave Carlin asked for this so maybe he should give his formatting desires, along with other ops people.  After LUA configuration is in, maybe we could look at configurable delimiters for this.  Adding the commas  between fields would not be that hard.

Reply:
Not sure there is reason to look at small details at this point, as this PR seems to be dead in the water until at least after the summit.

Reply:
Yeah, that's in interesting point. I had presumed escaping, but that's usually done by a parallel tag such as "ecqah" ('e' for 'escaped' / URI encoded). I think we'll do that in another PR.